charts/longhorn/100.1.1+up1.2.3/Chart.yaml

```diff
@@ -2,11 +2,11 @@ annotations:
   catalog.cattle.io/auto-install: longhorn-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Longhorn
-  catalog.cattle.io/kube-version: '>=1.18.0-0'
+  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.23.0-0'
```

Comment:
This should be '>= 1.18.0-0 < 1.25.0-0', the same as 1.3.0, and also for other 1.2.x charts. 

cc @PhanLe1010

Reply:
Perfect, I'll update all of Longhorn's 1.2.x to < 1.25 instead.